docs/guides/test-capability.md

```diff
@@ -77,99 +56,131 @@ describe('scope/profile-name/provider', () => {
 });
 ```
 
-<details>
+:::caution
+This test example will hit live APIs.
+
+If your capabilities require authorization, you can load keys from enviroment variables as described in [Run Capability](./run-capability#set-environment-variables)
+
+If you want to reduce amount of calls to live APIs, see section about [recording traffic](#recording-traffic).
+:::
+
+### Asserting results
 
-<summary>typescript example with typed client</summary>
+Usually we want to assert result coming from our map. Result from perform always returns a `Result` type that is either `Ok` or `Err`. This follows the [neverthrow](https://github.com/supermacro/neverthrow) approach.
```

Comment:
Just a sidenote, this is superuseful information which should be covered somewhere else in the docs. Maybe it's already in OneSDK docs though. Keep it here for now.

Reply:
OneSDK readme contains this partially on Github.